source/PlanetPanel.cpp

```diff
@@ -356,6 +364,11 @@ void PlanetPanel::CheckWarningsAndTakeOff()
 			if(missionCargoToSell > 0)
 				out << "cargo space to hold " << Format::CargoString(missionCargoToSell, "your mission cargo") << ".";
 		}
+		// Warn about unique items you sold.
+		else if(hasUniques)
```

Comment:
Not something to worry about for this PR, but am I correct in looking at this code and seeing that you can only get one of these four warnings? I'd hate to be the guy who hits continue on this dialog because I'll lose a job I didn't care about for not having enough cargo space and also end up losing a unique item I forgot I left in the outfitter that I wasn't warned about.

Perhaps loss of unique outfits should be the highest priority warning, but I'd prefer to be warned about losing an important storyline mission over losing a unique outfit, hence the desire to have multiple warnings be capable of displaying.

Reply:
The messages don't seem long, so it shouldn't be a problem to display all of them in a single dialog box.

Reply:
I'd wait with this change for #7645